src/main/java/io/obswebsocket/community/client/message/request/filters/CreateSourceFilterRequest.java

```diff
@@ -13,16 +13,16 @@
     private final Data requestData;
 
     @Builder
-    private CreateSourceFilterRequest(String sourceName, String filterName, Integer filterIndex, String filterKind, JsonObject filterSettings) {
+    private CreateSourceFilterRequest(String sourceName, String filterName, Boolean filterEnabled, Integer filterIndex, String filterKind, JsonObject filterSettings) {
```

Comment:
I don't see any filterEnabled param for this request in the Google Sheet

Reply:
oh, then i will  remove it. it was in the Data object so i just added it to the constructor